docs/deep-dives/data/sizing.md

```diff
@@ -0,0 +1,115 @@
+
+### Commentary on Exposure Rates
+
+Experiments often have 2 goals: 1) to measure the impact on users exposed to the treatment and 2) to quantify the impact on business KPIs. The exposure rate (% of enrolled clients who actually saw the treatment) affects both of these considerations. The exposure funnel proportionally reduces the number of users in the sample, thus reducing the power. Similarly, the smaller the exposure rate, the higher an impact on exposed users would be required to move business KPIs. One corollary to this is that risk is proportionally reduced. As an example, for a 1% exposure rate, we can run a much larger experiment for a given risk level (absolute number of users exposed). If exposure rates are very low, opportunity size is low but so is risk.
+
+## Understand Sizing Drivers
+
+There are often important business considerations which can effect the choice of sizing. These can include:
+
+1. Learning Goal
+2. Risks
+3. Opportunity size
+4. Prioritization
+5. Contractual requirements
+
+In many situations, these concerns dwarf statistical considerations.
+
+### Learning Goal
+
+What is the goal of the experiment?
+
+- Measure realized value, if any, from a low risk, high priority feature release. In this case, we can choose a large size or even consider a holdback design.
+- Determine opportunity size. In this case, we can run a small experiment to assess the opportunity size (exposure rate).
+- Demonstrate feature viability. In this case, statistical power is important, but we can also consider staged designs (see below) which maximize precision while reducing risk, at the expense of additional time/effort. We should strongly consider looking at upstream metrics to validate the causal hypothesis.
+- Prove a positive impact on guardrail metrics. In this case, a large experiment may be required.
+
+## Helpful Design Patterns
+
+In this section we introduce a few ways to string together sequences of experiments for various beneficial outcomes such as reducing risk or maximizing realized value.
+
+### Sequentially Larger
+
+In this pattern, we can run a sequence of experiments (2 or more) of increasing size. Some examples include:
+
+1. A small experiment (<5%) followed by a very large experiment (20%-100%).
+   - Risk is mitigated by running a small experiment (which will bound potential negative impacts) before exposing a large portion of users.
+   - Statistical power is maximalized by the very large experiment, giving the highest chance of measuring an impact.
+2. A very small experiment (<1%), followed by a small experiment (<5%), followed by a very large experiment.
+   - Similar to above, but further limits the risk of the 5% experiment through the use of a very small, preliminary experiment. See for example the rollout of ToU which used a 4-stage design.
```

Comment:
Can we include a link or links to help show this 4-stage design? Not a big deal if links don't exist or you don't think they'd be helpful.